web/src/routes/LicensePage.tsx

```diff
@@ -21,26 +21,21 @@ type PropsType = {
   onPress: string
 }
 
-const LicenseItem = (props: PropsType): ReactElement => {
-  const { name, version, license, onPress } = props
-  return (
-    <ListItem path={onPress} title={name}>
-      <LicenseContainer>
-        {`version: ${version}`}
-        <br />
-        {`license: ${license}`}
-      </LicenseContainer>
-    </ListItem>
-  )
-}
+const LicenseItem = ({ license, name, onPress, version }: PropsType): ReactElement => (
+  <ListItem path={onPress} title={name}>
+    <LicenseContainer>
+      <div>{`version: ${version}`}</div>
+      <div>{`license: ${license}`}</div>
+    </LicenseContainer>
+  </ListItem>
+)
 
 const LicensePage = (): ReactElement => {
```

Comment:
Should be the following (same for file name):
```suggestion
const LicensesPage = (): ReactElement => {
```